### DISCUSSION TITLE
Radar detection & scanny shit & Ect : a TLDR story

### DISCUSSION THREAD

**Discussion by Drillz007** [tsunamayo/Starship-EVO#4629] [Suggestions]
Sorry for the cluster of text in advance i hope its organized enough to make sense.

First idea i wanted to bring up was for detection was a cone of detection instead of a spherical detection, and for larger "antenna" to have a longer detection range but have less sensitivity so smaller vessels might go unnoticed. 
The reason i would like to see this in game is to give more types of interactions between different vessels based on detection and for multiple antenna on a single vessel instead of them being purely cosmetic (many other games have the issue of one and done).The way this can be done is by letting antenna be built up to judge the distance and size of what a ship is able to detect a smaller antenna would be able to pick up on all sorts of things but at the cost of being at a closer range a larger antenna would be able to pick up on things at very far distances but things the size of stations, shipyards, asteroids, or large ships. Reasons why this would be good is to help chop up and organize range for stuff to appear on screen as detected instead of it being cluttered all at once and create blind spots for large ships so smaller ships can be a bit more stealthy instead of being easy targets and hunted. Another thing to tac onto this is that if a ship flies without its shields on it has a lower detection signature as opposed to it flying with its bright high energy shields up.

Second idea (highly opinionated) has to do with shields 
Many other games have shields as the go to source of damage mitigation you stack them high go into a fight walk away without any damage. I personally feel that this is not the right way to go about this combat should be high risk high reward give and take and many shields offer full coverage at only the cost of only energy which is a basically free in comparison to taking hull damage. Suggestion for fixing this is to simply reduce the amount health shields either half standard hull/armor blocks HP have OR drastically increase the mass that shield generators to at least be twice the weight of a armor block. This allow shields to still reduce incoming damage but not be as tanky as other games.

Final idea had to do with having multiple types of thrusters 
This would be split into 4 groups the first group would be general purpose good speed average acceleration and good movement of mass.
Another one would be geared more towards top speed and high cruising with lesser ability to accelerate large mass.
Next would be a bigger high acceleration engine with a low top speed to make something like a cargo hauler less of a pain to maneuver around.
Last but not least would be the booster big thrusters with combined performance of the previous 3 but would have to be manually activated and have a large power drain to balance the performance so it would only really be able to be used when an emergency boost of power is necessary.

Thoughts comment concerns discussion is welcome.

**Comment by ZachZent** [comment]
https://github.com/tsunamayo/Starship-EVO/issues/1355 For scanning
